src/slam/include/slam_solver/graph_slam_solver/graph_slam_instance.hpp

```diff
@@ -93,6 +93,15 @@ class GraphSLAMInstance {
    */
   void process_observations(const ObservationData& observation_data);
 
+  /**
+   * @brief Loads a map and an initial pose into the graph
+   *
+   * @param map Coordinates of the map landmarks in the form of an Eigen vector [x1, y1, x2, y2,
+   * ...]
+   * @param pose Initial pose of the robot in the form of an Eigen vector [x, y, theta]
+   */
+  void load_map(const Eigen::VectorXd& map, const Eigen::VectorXd& pose);
```

Comment:
I would suggest making this function a bit more generic in terms of title, like load initial state or something like that. I would also reflect on receiving as argument the certainty of the map and pose of this initial estimate and not just fetch from parameters. I know it has access to it, this is just a suggestion